src/main/java/Kitty.java

```diff
@@ -97,9 +110,12 @@ public static void markTask(String line) {
 
     private static boolean isTaskNumValid(String line) {
```

Comment:
This may be confused for a boolean variable instead of a method. Maybe use getTaskNumValidity() as a name instead.